mixserv/src/main/java/hivemall/mix/server/MixServer.java

```diff
@@ -94,6 +102,20 @@ static Options getOptions() {
         return opts;
     }
 
+    @Override
+    public String toString() {
+        final List<String> configs = new ArrayList<String>();
```

Comment:
Okay though, this function is used in a pr #236 multiple times;
https://github.com/myui/hivemall/pull/236/files#diff-2a9c23421000c9d32f8d2756657aed5bR23

We also need to fix the codes in #236 in this way?